frontend/public/components/routes.tsx

```diff
@@ -85,12 +85,89 @@ export const RouteHostname: React.SFC<RouteHostnameProps> = ({obj}) => <div>
   }
 </div>;
 
+export const RouteStatus: React.SFC<RouteHostnameProps> = ({obj: route}) => {
+  let admittedCondition: boolean = false;
+
+  const statusIconStyle = {
+    paddingRight: '6px',
+    width: '16px'
+  };
+
+  if (!route.status || !route.status.ingress) {
+    return <span>
+      <div className="fa fa-hourglass-half" style={statusIconStyle} aria-hidden="true"></div>
+      Pending
+    </span>;
+  }
+
+  _.each(route.status.ingress, (ingress) => {
+    let condition: any = _.find(ingress.conditions, {type: 'Admitted', status: 'True'});
+    if (condition) {
+      admittedCondition = true;
+    }
+  });
+
+  return <React.Fragment>
+    {admittedCondition && <span>
+      <div className="fa fa-check text-success" style={statusIconStyle} aria-hidden="true"></div>
+      Accepted
+    </span>}
+    {!admittedCondition && <span>
+      <div className="fa fa-times text-danger" style={statusIconStyle} aria-hidden="true"></div>
+      Rejected
+    </span>}
+  </React.Fragment>;
+};
+
+const addTLSWarnings = (route: any, warnings: string[]) => {
```

Comment:
I'm still on the fence about adding all these warnings. I think these problems are uncomment and we should focus on showing status conditions only.